Soccar [Library]/src/nl/soccar/library/enumeration/HandbrakeAction.java

```diff
@@ -2,11 +2,33 @@
 
 /**
  * Enumeration of all available steer actions.
- * 
+ *
  * @author PTS34A
  */
 public enum HandbrakeAction {
+
+    ACTIVE(0),
+    INACTIVE(1);
+    
+    private final int id;
     
-    ACTIVE, INACTIVE
+    private HandbrakeAction(int id) {
+        this.id = id;
+    }
     
+    public int getId() {
+        return id;
+    }
+    
+    public static HandbrakeAction parse(int id) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)